core/server/apps/subscribers/lib/helpers/subscribe_form.js

```diff
@@ -43,6 +43,7 @@ subscribeScript =
 subscribe_form = function (options) {
     var root = options.data.root,
         data = _.merge({}, options.hash, _.pick(root, params), {
+            // @TODO: shall we use `urlJoin` here instead of `path.join`?
```

Comment:
Is this something, we should do globally? Found some more places where we could use `urlJoin` instead of `path.join`...